broken-site-reporting/multiple_report_tests.json

```diff
@@ -23,7 +24,8 @@
                             {"name": "tds", "value": "abc123"},
                             {"name": "blockedTrackers", "value": "bad.tracker.test,tracking.test"},
                             {"name": "surrogates", "value": "surrogate.domain.test,domain2.test"},
-                            {"name": "lastSentDay", "present": false}
+                            {"name": "lastSentDay", "present": false},
+                            {"name": "protectionsState", "value": "1"}
```

Comment:
oh boy, we have `"1"`, `"true"` and `false` in the params list 😬 Can we still make `protectionsState` a proper boolean?

Reply:
it's already implemented across all platforms as `"1" | "0"` - but it would be trivial to change it now if we'd prefer? Probably better for me to update platforms now, as part of this project if we regret the "1" | "0".

I think I was under the impression the choice was discussed etc, I could've/should've questioned it perhaps.